depends/packages/openssl.mk

```diff
@@ -57,7 +57,9 @@ $(package)_config_opts_riscv64_linux=linux-generic64
 $(package)_config_opts_x86_64_darwin=darwin64-x86_64-cc
 $(package)_config_opts_x86_64_mingw32=mingw64
 $(package)_config_opts_i686_mingw32=mingw
-$(package)_config_opts_android=linux-generic64
+$(package)_config_opts_aarch64_android=linux-generic64 -fPIC
+$(package)_config_opts_armv7a_android=linux-generic32 -fPIC
+$(package)_config_opts_i686_android=linux-generic32 -fPIC
```

Comment:
Missing x86_64 here as well.

Reply:
I pushed the changes

Reply:
I don't think it's a big deal, let's leave as is, one less line

Reply:
OK, so i'll leave it as is now.